eva-accession-clustering/src/test/java/uk/ac/ebi/eva/accession/clustering/batch/io/clustering_writer/ReuseAccessionClusteringWriterTest.java

```diff
@@ -224,6 +231,7 @@ public void reuse_eva_clustered_accession_when_clustering_a_dbsnp_submitted_vari
         assertEquals(0, mongoTemplate.count(new Query(), DbsnpClusteredVariantEntity.class));
         assertEquals(0, mongoTemplate.count(new Query(), SubmittedVariantEntity.class));
         assertEquals(1, mongoTemplate.count(new Query(), ClusteredVariantEntity.class));
+        assertEquals(0, mongoTemplate.count(new Query(), SubmittedVariantOperationEntity.class));
```

Comment:
The idea with this asserts is to check the DB status before and after the operations are performed so it is good to check both operation collections eva and dbsnp. Specially in this case where the target collection will be the dbsnp.
```suggestion
        assertEquals(0, mongoTemplate.count(new Query(), DbsnpSubmittedVariantOperationEntity.class));
        assertEquals(0, mongoTemplate.count(new Query(), SubmittedVariantOperationEntity.class));
```
Please chech this in other test methods too

Reply:
Added the suggested asserts.